config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_secretmanagersecretversions.secretmanager.cnrm.cloud.google.com.yaml

```diff
@@ -170,23 +177,20 @@ spec:
                   type: object
                 type: array
               createTime:
-                description: Output only. The time at which the [SecretVersion][google.cloud.secretmanager.v1.SecretVersion]
-                  was created.
+                description: 'DEPRECATING NOTE: Please use status.observedState.createTime
```

Comment:
I'm a little unsure about deprecating these fields.  It feels like there's a lot of things happening in this PR!

Reply:
Right. there is. We do not actually deprecate this field (it's "deprecating" 😃 ). Maybe we will deprecate it in v1 version. But I want to start adding this note and see what users say in OSS.

Reply:
OK, as long as we are still populating the values.  I do think we could phrase this better - I think it is a deprecation.  I think maybe something like "Deprecated.  Please use the corresponding field in status.observedState.  This field will continue to exist and be populated in the v1beta1 API, but may be removed in a future API revision in favor of status.observedState."  (I don't think we need to include the field name)  We can do that in another PR.